src/build/env.py

```diff
@@ -130,7 +130,7 @@ def __enter__(self):  # type: () -> IsolatedEnvironment
 
         self._symlink_relative(sysconfig.get_path('include'))
         self._symlink_relative(sysconfig.get_path('platinclude'))
-        self._symlink_relative(sysconfig.get_config_var('srcdir'))
+        self._symlink_relative(sysconfig.get_config_var('LIBPL'))
```

Comment:
also we could document why this is needed 🤷

Reply:
Commit description

> LIBPL contains the files necessary for embedding Python in C, I wrongly
assumed srcdir was used for this. srcdir will point to LIBPL in Python
3, but not in Python2.

Reply:
Ah, I see. Often I make it a code comment, as the commit history gets lost easier.